core/src/main/java/com/orientechnologies/orient/core/storage/impl/local/paginated/atomicoperations/OAtomicOperation.java

```diff
@@ -52,6 +52,7 @@
   private Map<String, Long>      newFileNamesId       = new HashMap<String, Long>();
   private Set<Long>              deletedFiles         = new HashSet<Long>();
   private Map<String, Long>      deletedFileNameIdMap = new HashMap<String, Long>();
+  private List<OWALRecord>       externalLogs         = new ArrayList<OWALRecord>();
```

Comment:
That is for develop branch as I can see, could you use diamond operator instead  ?))

Reply:
:)